### PR TITLE
feat: add generator package for updating package.json files consistently

### LINKED ISSUE #1641: Packages are missing the repository field in package.json
The `repository` field is important to indicate where the code/package lives, especially in a monorepo.
See for example https://github.com/commercetools/merchant-center-application-kit/blob/8537a38431c428b245a8d178651d1d90efcbb0e9/packages/application-shell/package.json#L6-L10

### PR DESCRIPTION
**Pull request by emmenko**
Closes #1641

### CODE REVIEW

File: generators/package-json/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  return omitEmpty({
+    name: `${npmScope}/${packageFolderName}`,
+    description: originalPackageJson.description,
+    version: originalPackageJson.version,
+    bugs: 'https://github.com/commercetools/ui-kit/issues',
+    repository: {
+      type: 'git',
+      url: 'https://github.com/commercetools/ui-kit.git',
+      directory: relativePackageFolderPath,
```
Comment (emmenko) [inline generators/package-json/src/index.ts:62]: This part here is nice to have it autogenerated.

File: husky.config.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 module.exports = {
   hooks: {
     'commit-msg': 'commitlint -e $HUSKY_GIT_PARAMS',
-    'pre-commit': 'lint-staged',
+    'pre-commit':
+      'lint-staged && yarn generate-package-json --all-workspace-packages',
```
Comment (emmenko) [inline husky.config.js:5]: We can run the generator on each commit, it just takes a couple of sec.

**Comment by emmenko** [comment]
We have 47 packages that are missing the `description` field:

```
Error: Found errors in 47 packages:

Package design-system is missing the description field
Package packages/calendar-time-utils is missing the description field
Package packages/calendar-utils is missing the description field
Package packages/components/buttons/accessible-button is missing the description field
Package packages/components/buttons/flat-button is missing the description field
Package packages/components/buttons/icon-button is missing the description field
Package packages/components/buttons/link-button is missing the description field
Package packages/components/buttons/primary-button is missing the description field
Package packages/components/buttons/secondary-button is missing the description field
Package packages/components/buttons/secondary-icon-button is missing the description field
Package packages/components/collapsible-motion is missing the description field
Package packages/components/collapsible-panel is missing the description field
Package packages/components/constraints is missing the description field
Package packages/components/data-table-manager is missing the description field
Package packages/components/field-errors is missing the description field
Package packages/components/field-label is missing the description field
Package packages/components/grid is missing the description field
Package packages/components/icons is missing the description field
Package packages/components/inputs/async-select-input is missing the description field
Package packages/components/inputs/checkbox-input is missing the description field
Package packages/components/inputs/localized-money-input is missing the description field
Package packages/components/inputs/localized-rich-text-input is missing the description field
Package packages/components/inputs/radio-input is missing the description field
Package packages/components/inputs/rich-text-input is missing the description field
Package packages/components/inputs/select-utils is missing the description field
Package packages/components/inputs/time-input is missing the description field
Package packages/components/inputs/toggle-input is missing the description field
Package packages/components/label is missing the description field
Package packages/components/link is missing the description field
Package packages/components/messages is missing the description field
Package packages/components/notifications is missing the description field
Package packages/components/primary-action-dropdown is missing the description field
Package packages/components/spacings/inline is missing the description field
Package packages/components/spacings/inset is missing the description field
Package packages/components/spacings/inset-squish is missing the description field
Package packages/components/spacings/stack is missing the description field
Package packages/components/table is missing the description field
Package packages/components/tag is missing the description field
Package packages/components/text is missing the description field
Package packages/hooks is missing the description field
Package packages/i18n is missing the description field
Package packages/localized-utils is missing the description field
Package presets/buttons is missing the description field
Package presets/fields is missing the description field
Package presets/inputs is missing the description field
Package presets/spacings is missing the description field
Package presets/ui-kit is missing the description field
```

File: design-system/package.json [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
   "version": "10.18.4",
-  "description": "",
+  "bugs": "https://github.com/commercetools/ui-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/ui-kit.git",
+    "directory": "design-system"
+  },
+  "homepage": "https://uikit.commercetools.com",
+  "keywords": ["javascript", "design system", "react", "uikit"],
```
Comment (adnasa) [inline design-system/package.json:12]: what about "component library"?

**Review by adnasa — APPROVED** [review]
this is really cool.
Well done 👌🏽

**Review by jonnybel — APPROVED** [review]
Nice, thanks 👍

**Review by tdeekens — APPROVED** [review]

**Pull request merged**